src/peergos/shared/user/fs/RetrievedCapability.java

```diff
@@ -1,57 +1,63 @@
 package peergos.shared.user.fs;
 
-import peergos.shared.cbor.CborObject;
```

Comment:
did you mena to update the permissions of `` src/peergos/shared/user/fs/RetrievedCapability.java``?

Reply:
Not sure what you mean?